recipes-core/initrdscripts/tegra-flash-init_1.0.bb

```diff
@@ -14,6 +14,9 @@ S = "${WORKDIR}"
 
 do_install() {
     install -m 0755 ${WORKDIR}/init-flash.sh ${D}/init
+    if [ -f "${EXTRA_INIT}" ]; then
```

Comment:
It wouldn't hurt to add a default initializer for this variable at the top of the recipe, i.e.
```
EXTRA_INIT ??= ""
```

Reply:
That's a good idea, but I'd move the logic into the `init_extra` script itself, which we could supply with the recipe, and users could just install their customizations into a directory, say, `/init_extra.d`.  That would eliminate the need to add a bbappend to this recipe altogether - the recipes to install the custom extension scripts could be separate, and would just need to be included in the `tegra-initrd-flash-initramfs` image. We already have a `TEGRA_INITRD_FLASH_INSTALL` variable (which defaults to empty) that could be set in a distro config file to add the custom extension packages that install those scripts.

Reply:
Good input. I'll get on it and update the PR.